### PR TITLE
IJSDK-533: DialogWrapper example

### PR DESCRIPTION
**Pull request by a-zen**
I created some simple code examples for the DialogWrapper

https://youtrack.jetbrains.com/issue/IJSDK-533

### CODE REVIEW

File: user_interface_components/dialog_wrapper.md [hunk 1]
```diff
@@ -70,3 +70,50 @@ Otherwise, you need to return a
...
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class SampleDialogWrapper extends DialogWrapper {
+
+    public SampleDialogWrapper() {
+        super(true); // use current window as parent
+        super.init();
```
Comment (yole) [inline user_interface_components/dialog_wrapper.md]: No need to use an explicit "super" qualifier.

File: user_interface_components/dialog_wrapper.md [hunk 2]
```diff
@@ -70,3 +70,50 @@ Otherwise, you need to return a
...
+        super.setTitle("Test DialogWrapper");
+    }
+
+    @Nullable
+    @Override
+    protected JComponent createCenterPanel() {
+        JPanel dialogPanel = new JPanel(new BorderLayout());
+
+        JLabel label = new JLabel("testing");
+        dialogPanel.add(label);
```
Comment (yole) [inline user_interface_components/dialog_wrapper.md]: When using `BorderLayout`, it's better to specify explicit constraints.

File: user_interface_components/dialog_wrapper.md (hunk 1 above)
Comment (a-zen) [reply user_interface_components/dialog_wrapper.md]: you're right thx

File: user_interface_components/dialog_wrapper.md (hunk 2 above)
Comment (a-zen) [reply user_interface_components/dialog_wrapper.md]: I added prefered size and explicitly positioned the label. hope this is what you meant. ;-)

File: user_interface_components/dialog_wrapper.md [hunk 3]
```diff
@@ -70,3 +70,50 @@ Otherwise, you need to return a
 class which encapsulates an error message and an optional component associated with the invalid data.
 If you specify a component, an error icon will be displayed next to it, and it will be focused when the user tries to invoke the `OK` action.
 
+## Example
+
+Simple definition of a
+[DialogWrapper](upsource:///platform/platform-api/src/com/intellij/openapi/ui/DialogWrapper.java):
+
+```java
+package com.example.plugin.component;
```
Comment (YannCebron) [inline user_interface_components/dialog_wrapper.md]: no package names or imports in code samples, please

Comment (a-zen) [reply user_interface_components/dialog_wrapper.md]: ok fix this.

**Comment by YannCebron** [comment]
@yole AFAIU your request changes have been applied?

**Pull request merged**